CONTRIBUTING.md

```diff
@@ -322,3 +324,22 @@ precompile/build-all-addons.js
 ```
 
 NOTE: It is recommended to comment out the ABI versions you don't want to generate/regenerate in the `build-all-addons` script. Alternatively, if you want to regenerate all builds that's fine too, but it takes longer.
+
+#### How to publish local layer to AWS
+
+To publish local layer to AWS with the AWS CLI, the CLI needs to have access to the IAM credentials.
+
+Steps to follow
+
+- Create an IAM user if you don't already have one with the required rights.
+- Create `credentials` file on the following folder location, `/usr/local/aws`
+- Add below informations to the newly created `credentials` file. The access key and secret key can be obtained from the IAM user account
+```javascript
+  - [default]
+  - aws_access_key_id = <add your access key>
+  - aws_secret_access_key = <add your secert key>
+```
```

Comment:
See https://github.com/instana/nodejs/blob/ae36337a926efec4cb38b961959e37d4227dcd08/CONTRIBUTING.md#how-to-publish-local-layer-to-aws

I think you need to remove hyphens.